koku/masu/processor/orchestrator.py

```diff
@@ -67,12 +67,18 @@ def __init__(
         self.queue_name = queue_name
         self.ingress_reports = kwargs.get("ingress_reports")
         self.ingress_report_uuid = kwargs.get("ingress_report_uuid")
-        self._polling_accounts = Provider.polling_objects.get_accounts()
         self._summarize_reports = kwargs.get("summarize_reports", True)
 
     def get_polling_batch(self):
+        if not self.scheduled and self.provider_uuid:
+            providers = Provider.objects.filter(uuid=self.provider_uuid)
+        else:
+            filters = {}
+            if self.provider_type:
+                filters["type"] = self.provider_type
```

Comment:
setting this filter allows us to filter on the provider-type and respect the polling-timestamps by using the `polling_objects` Manager.